src/ZoneServer/Network/Send.cs

```diff
@@ -2031,6 +2031,8 @@ public static void ZC_MAP_REVEAL_LIST(IZoneConnection conn)
 			{
 				packet.PutInt(revealedMap.MapId);
 				packet.PutBin(revealedMap.Explored);
+				packet.PutFloat(revealedMap.Percentage);
+				packet.PutFloat(0);				
```

Comment:
Have you perchance gained any new insights into what this second float does? Which we apparently documented in the binary templates, but never gave a proper name.

Speaking of binary templates, since the identity of the first float appears clear now, it might be a good idea to also update the templates to reflect this, to improve maintainability and debuggability. Again, not something I would enforce, but a nice to have.

Reply:
I wasn't aware that we had templates, you'll have to let me know where they are some time.

As for the second Float, I have no idea what it does, it might even just be a spacer.